recipes/us/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{%set name = "us" %}
+{%set version = "0.9.1" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "ed374a332f6e1d9cec87a44ebd2404f3a4383c6a77297adb259344a353af3c92" %}
+{%set build_num = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  entry_points:
+    - states = us.cli.states:main
+
+  number: {{ build_num }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
Please add the [entry point]( http://conda.pydata.org/docs/building/meta-yaml.html#python-entry-points ).

```yaml
  entry_points:
    - states = us.cli.states:main
```